rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -1268,6 +1268,19 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 			m_gui_settings->SetValue(gui::gs_showMouseFs, val);
 		});
 
+		ui->gs_hideMouseOnIdle->setChecked(m_gui_settings->GetValue(gui::gs_hideMouseIdle).toBool());
+		connect(ui->gs_hideMouseOnIdle, &QCheckBox::clicked, [this](bool val)
+		{
+			m_gui_settings->SetValue(gui::gs_hideMouseIdle, val);
+			ui->gs_hideMouseOnIdleTime->setEnabled(val);
+		});
+		ui->gs_hideMouseOnIdleTime->setEnabled(m_gui_settings->GetValue(gui::gs_hideMouseIdle).toBool());
```

Comment:
no need to call the settings again. you already have the value in the gs_hideMouseOnIdle checkbox